java/src/jmri/jmrix/openlcb/OlcbConfigurationManager.java

```diff
@@ -467,18 +469,18 @@ private void  addStringPart(String value, List<Byte> contents) {
         SimpleNodeIdentInfoHandler() {
             List<Byte> l = new ArrayList<>(256);
             l.add((byte)4); // version byte
-            addStringPart("JMRI", l);
-            addStringPart("PanelPro", l);
+            addStringPart("JMRI", l, 40);
```

Comment:
No need to change, but a suggestion.

It might be good to add a comment like:
`// The standard tells no more than 41 bytes including terminating null.`
`// The standard tells no more than 64 bytes including terminating null.`
That explains why these numbers differs by one from the standard.

It's nothing important, but removes the risk of someone later reads the standard, compares the values here, and thinks the values here are wrong and therefore "correct" them.